pkgs/development/python-modules/numpy.nix

```diff
@@ -12,6 +12,12 @@ in buildPythonPackage (args // rec {
   buildInputs = args.buildInputs or [ gfortran nose ];
   propagatedBuildInputs = args.propagatedBuildInputs or [ passthru.blas ];
 
+  patches = lib.optionals isPy27 [
+    # See cpython 2.7 patches.
+    # numpy.distutils is used by cython during it's check phase
+    ./numpy-distutils-C++.patch
```

Comment:
I made this patch myself. I don't see the point of reporting the issue before the main distutils patch is merged to python2, which won't happen.